### PR TITLE
Support Tutor 15 and Open edX Olive

### PR DESCRIPTION
**Pull request by mrtmm**

### CODE REVIEW

**Comment by fghaas** [comment]
This broke in CI because it's still trying to run on Python 3.6, which really isn't needed. We need to land #18 first and then we can rebase this.

When you do, please split the `README.md` update into a separate commit, like we did in https://github.com/hastexo/tutor-contrib-enrollmentreports/pull/18.

File: tutoropenstack/plugin.py [hunk 1]
```diff
@@ -27,12 +27,7 @@
     },
 }
 
-hooks.filters.add_items(
-    "cli:commands",
-    [
-        openstack_command,
-    ],
-)
+hooks.Filters.CLI_COMMANDS.add_item(openstack_command)
```
Comment (fghaas) [inline tutoropenstack/plugin.py:30]: So just checking: do we know that this is a non-breaking, backwards-compatible change that doesn't at all change the minimum Tutor version required? In other words, will this really work with _any_ Nutmeg Tutor version all the way back to 14.0.0?

**Review by fghaas — APPROVED** [review]
This looks good to me now — should I go ahead and merge, and create the `nutmeg` branch?

**Comment by mrtmm** [comment]
Oh wait, I think we don't _need_ the nutmeg branch here.

**Comment by fghaas** [comment]
Yes, you're right. :)

**Pull request merged**